README.md

```diff
@@ -174,6 +180,13 @@ all cases, but for the 2NT bids we use Puppet Stayman.
 * 3♣️! Natural, signoff
 * 3♦️! Invitation to 3NT. Requires long diamonds (6+) as opener passes with
   minimum.
+* 3♥️ Invite to 4♥️, guarantees 5+♥️s
+* 3♠️ Invite to 4♠️, guarantees 5+♠️s
+* 3NT To play
+* 4♥️ To play
+* 4♠️ To play
+* 5♣️ To play
```

Comment:
Is 4NT still King ask?

Reply:
Not sure what you mean by king ask. If you meant is it RKC, then we don't really have a trump suit. I think there are two reasonable options here:

(1) With lack of a clear trump suit after a 2D opener, RKC is in clubs (since it's 4-5 where the majors are 3-4)
(2) Use Blackwood (only count aces).

Reply:
Whoops I meant Ace ask. Probably just do Blackwood to reduce confusion of the suit for King.